README.md

```diff
@@ -29,6 +29,11 @@ Usage of ./appuio-keycloak-adapter:
       The address of the Keycloak server (E.g. https://keycloak.example.com).
   -keycloak-username string
       The username to log in to the Keycloak server.
+
+  -sync-schedule string
+      A cron style schedule for the organization synchronization interval. (default "@every 5m")
```

Comment:
I probably would have gone with a K8s CronJob instead of running cron inside the app. However, with a default of 5 minute interval, K8s CronJobs aren't the best fit either.

Reply:
Yes I wasn't sure how to properly do this either. As IMO the import should run quite frequently I opted to do it as part of the adapter binary, similarly to the group-sync-operator. Not entirely sure if this is the best approach, but I think I would leave it like that for now.